test/unit/utils/SceneUtils.js

```diff
@@ -0,0 +1,43 @@
+/* global QUnit */
+
+import { Object3D } from '../../../../src/core/Object3D.js';
+
+export default QUnit.module( 'Utils', () => {
```

Comment:
This unit test is not executed since it is not included in `three.source.unit.js`. This file is for `src` modules only so it best to delete `test/unit/utils/SceneUtils.js` altogether.

Side note: Since the methods were moved to `SceneUtils`, code like `a.traverseGenerator()` is not valid. So the current implementation of the unit test would fail in any event.

Reply:
My bad you're right, I was a bit hasty with it to be quite frank

Reply:
Side-note but is the E2E testing on every commit necessary? I'm not well-versed in CI but it feels like something that should be manually triggered given how much github action time it must take.